### PR TITLE
Describe "Control feature rendering order" option

### PR DESCRIPTION
**Pull request by DelazJ**
fix partially #736 (might be worth adding a screenshot)

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -672,9 +672,29 @@ features of the layer:
...
+  Activate the checkbox and click on the |browsebutton| button beside.
+  You then get the :guilabel`Define Order` dialog in which you:
+
+  * choose a field or build an expression to apply to the layer features
+  * set in which order the fetched features should be sorted, i.e. if you choose 
+    **Ascending** order, the features with lower value are rendered under those
+    with upper value.
+  * define how NULL values should be handled: **NULLs first** means that the features
+    that have NULL value for this field or for which the expression returns NULL
+    are drawn below all the other features. **NULLs last** places them over all.
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: do we need to explain in such details something so obvious? :)

Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Are you talking about the nulls first/nulls last options only or about the whole decription of the process?
About Null handling, I wasn't sure if first meant below or above reason why I explained. But I'm free to fix it if you point out what seems obvious :)

Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: yes, I meant only **NULLs first**/**NULLs Last** seems obvious to me. I just wondered if it is a good thing to add such details as time moving we will get a documentation growing too much with too much details.

**Comment by DelazJ** [comment]
@yjacolin I shorten the explanation and personally find it better. Hope you too :)
Will fix the conflicts when i'm on my local repository.

**Comment by yjacolin** [comment]
LGTM

**Pull request merged**